projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/schema/plugin.json

```diff
@@ -2,6 +2,20 @@
   "jupyter.lab.shortcuts": [],
   "title": "vdk-jupyterlab-extension",
   "description": "vdk-jupyterlab-extension settings.",
+  "jupyter.lab.menus": {
+    "main": [
+      {
+        "id": "jp-vdk-menu",
+        "label": "VDK",
+        "rank": 80,
```

Comment:
Is this the default value? This property is optional so maybe we can just remove it?

Reply:
yes, it is optional. it is about where the menu is located. With 80 we get sth between other menus